docs/Commands.md

```diff
@@ -863,9 +863,9 @@ FT.GET {index} {doc id}
 
 ### Description
 
-Returns the full contents of a document.
+!!! warning "This command is deprecated". Use HGETALL instead.
 
-Currently it is equivalent to HGETALL, but this is future-proof and will allow us to change the internal representation of documents inside Redis in the future. In addition, it allows simpler implementation of fetching documents in clustered mode.
+Returns the contents of a document as insereted but removes attribute fields(score/language/Payload) if present.
```

Comment:
[formatting]
Returns content of a document as inserted without attribute fields (score/language/payload).